gatsby-config.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
   siteMetadata: {
     title: 'Carbon Design System',
     titleInternal: 'IBM Product Design',
+    siteUrl: 'https://next.carbondesignsystem.com'
```

Comment:
should we wait and update this to `www.` and merge it in once we're live? or do we need it now